### ISSUE TITLE
Windows: support more package managers

### ISSUE THREAD

**Issue by Un1q32** [Gobidev/pfetch-rs#11]
it runs but theres no logo, os detection, or package count
memory, uptime, and kernel work though
![Screenshot 2023-02-24 152053](https://user-images.githubusercontent.com/57970114/221216621-1449911e-4fe3-4b82-a517-bc211c27a7d9.png)

**Comment by Gobidev** [comment]
I fixed os and host detection in ab02f3c39e87eab7013453c6d9d1368c971c2c91. There is currently no logo because pfetch does not have one for Windows. I will probably try to make one but my ASCII art skills are not the best. Package count could probably work with [`chokolatey`](https://chocolatey.org/) or [`winget`](https://github.com/microsoft/winget-cli), trying to count the packages installed otherwise would be too unreliable.

**Comment by Gobidev** [comment]
![image](https://user-images.githubusercontent.com/50576978/221260185-e4509a03-6763-4ff4-ab89-d3b4c0de0e4c.png)
I added a simple logo, this is what pfetch now looks like on Windows.

**Comment by Gobidev** [comment]
39785f191115c83321a7d4b6a8b916fc28361f96: I now use libmacchina for Windows package count, which currently supports `cargo` and `scoop`. I may try to add support for `winget` and `chocolatey` in the future.

**Comment by OldWorldOrdr** [comment]
i recommend using # for the windows logo instead of l

**Comment by Gobidev** [comment]
You are right, that looks better. As of 5d037c558f0c69e0d6e3430b5383ed91f29df969 it now looks like this:

![image](https://user-images.githubusercontent.com/50576978/221379194-b858552b-7b5b-4e5a-a59c-6192b6d89eae.png)

**Comment by Gobidev** [comment]
Added Windows binary in [v2.2.0](https://github.com/Gobidev/pfetch-rs/releases/tag/v2.2.0)

**Comment by OldWorldOrdr** [comment]
Could you add support for pacman with msys2?  I can't imagine it would be too difficult since pacman works with arch linux, and its the same on msys2

**Comment by Gobidev** [comment]
I did some testing and it is actually not that difficult, but it would requre a change in libmacchina and it is possible that this is too much of a niche usecase to be added. I will make a PR soon and we'll see.

**Comment by OldWorldOrdr** [comment]
I wouldn't call this niche.  MSYS2 is how people get any unix-like CLI things working on Windows without WSL (like if you run windows 8)
